### ISSUE TITLE
Browser window and nav bar should appear together when making a new window.

### ISSUE THREAD

**Issue by bluemarvin** [MozillaReality/FirefoxReality#1641]
## Configuration

Firefox Reality version: all
Firefox Reality build ID: all

Hardware: all

## Steps to Reproduce

1. Create a new window.

## Current Behavior

The nav bar is shown but the window is not show until gecko renders to the surface texture.
## Expected Behavior

A window should be shown even if it is blank.
## Possible Solution

Draw a temporary quad placeholder for the window. It could be white for regular windows and purple for private browsing. Not drawing the window immediately on window creation makes FxR seem unresponsive and slow. We would need to have different implementations for layers and non-layer window rendering.

**Comment by bluemarvin** [comment]
These GeckoView issues need to be resolved to unblock this feature.
https://bugzilla.mozilla.org/show_bug.cgi?id=1578947
https://bugzilla.mozilla.org/show_bug.cgi?id=1578949

**Comment by Softvision-GeluHaiduc** [comment]
I have verified that this issue is no longer reproducible with the latest Firefox Reality 1.5-32622233 (cfdab616) installed on the Oculus Go, Quest, HTC Vive Focus, Focus Plus and Lenovo Mirage Solo headsets.
Now, the browser window and nav bar appear together when a new window is opened, and a placeholder is displayed.
However, it seems that the color of the placeholder background is easily confused with the background of the max window limit popup. I logged a new issue regarding this behavior: #1862
